guidelines/terms/21/single-pointer.html

```diff
@@ -1,6 +1,6 @@
 <dt><dfn id="dfn-single-pointer">single pointer</dfn></dt>
 <dd>
-   <p>an input that only targets a single point on the page/screen at a time – such as a mouse, single finger on a touch screen, or stylus.</p>
-   <p class="note">In contrast to single pointer inputs, multipoint interactions involve the use of two or more pointers at the same time – such as two finger interactions on a touchscreen, or the simultaneous use of a mouse and stylus.</p>
+   <p>an input mechanism that only targets a single point on the page/screen at a time – such as a mouse, single finger on a touch screen, or stylus.</p>
```

Comment:
I think using "mechanism" makes this more confusing, not less so. WCAG has a definition of "mechanism", which doesn't seem to be the same thing that you're using the word for here.

The phrase that confused me was "at a time". The way I think of path-based gestures is that it is multiple inputs coming in at multiple times. So since it doesn't happen "at a time" but "over time", I figured the definition doesn't apply. After talking to @dbjorge I learned that what you meant here is that it can be over time, just not at the same time. It's that part that I think needs to be clarified.

I think that removing "single taps and clicks, double-taps and clicks, long presses, and path-based gestures" from this definition has made it more confusing. I think these need to be put back either in the definition text or in a note.

Reply:
but again, the pointer is not the gesture. the single pointer is the one finger on the touchscreen, the mouse, the stylus. you use pointers to then perform gestures. i can propose a further tweak on friday expanding the note, but this fundamental distinction needs to be finally put to bed (as it causes non-sequiturs like the one the original change addresses, which also caused confusion)

Reply:
I agree with Wilco that I think adding "mechanism" makes it more confusing, I'd prefer omitting it.

Reply:
```suggestion
   <p>an input modality that only targets a single point on the page/screen at a time – such as a mouse, single finger on a touch screen, or stylus.</p>
```